Integrations/integration-HybridAnalysis.yml

```diff
@@ -354,6 +363,186 @@ script:
+                    'Positives': key.positives
+                });
+            if (key.status == 'unknown' || key.status =='no-result' )
+                unknown_counter++;
+            else if (key.status == 'malicious')
+                malicious_counter++;
+       //     num_elements++;
+        });
+
+        //calculating DBot Score
+        if (unknown_counter == response.scanners.lenght)
+            Score = 0;
+        if (malicious_counter == 0)
+            Score = 1;
+        if (malicious_counter == 1)
+            Score = 2;
+        if (malicious_counter >= threshold)
+            Score = 3;
+
+        var Dbot_context = {
+            'Indicator': response.sha256,
+            'Vendor': "Hybrid Analysis",
+            'Score': Score,
+            }
+        var whitelist = {
+            'ID': response.whitelist[0].id,
+            'Value' : response.whitelist[0].value
+        }
+        var res = {};
+        if (Score == 3){
+            res.Malicious = {
+                Vendor: 'Hybrid Analysis',
+                Description: 'DBotScore is 3' // check this!
+            };
+        }
+
+        //create context
+        var file_context = {
+            'ID': response.id,
+            'SHA256': response.sha256,
+            'Scanner' : scanners,
+            'Finished': response.finished,
+            'Whitelist': whitelist
+        };
+
+        if (indicator == 'File') {
+            res.EntryID = response.id;
+            res.SHA256 = response.sha256;
+            Dbot_context.Type = 'File';
+            var context= {
+                'HybridAnalysis.File(val.id && val.id == obj.id)' : file_context,
+                'File(val.id && val.id == obj.id)' : res,
+                'DBotScore' : Dbot_context
+            };
+        }
+
+        else {
+            Dbot_context.Type = 'URL';
+            var context= {
+                'HybridAnalysis.URL(val.id && val.id == obj.id)' : file_context,
+                'DBotScore' : Dbot_context
+            }
+            // if (Score == 3)
+            //     context.URL.Malicious = res;
+        }
+
+
+        var md = '### Scan Results: \n ##### ID: ' + response.id + '\n' + '##### SHA256: ' + response.sha256 + '\n' + '##### finished: ' + response.finished + '\n' ;
+        var scanTable = '##### scanners:\n' + tableToMarkdown(null, scanners, headers, undefined, headerTransform=undefined, removeNull=true);
+        var whitelistTable = '##### whitelist:\n' + tableToMarkdown(null, whitelist, headers, undefined, headerTransform=undefined, removeNull=true);
+
+        hr = md + scanTable + '\n' + whitelistTable;
+
+        return {
+            Type: entryTypes.note,
+            ContentsFormat: formats.json,
+            Contents: response,
+            ReadableContentsFormat: formats.markdown,
+            HumanReadable: hr,
+            EntryContext: context
+        };
+    }
+
+
+    function quick_scan_to_entry(response, indicator, url) {
```

Comment:
might be clearer if we rename indicator to indicatorType.
Also, an explaining comment could come in handy here, since currently the function is complicated (since the product's logic is complicated inherently)

Reply:
Done